scripts/manage-imports.py

```diff
@@ -19,8 +20,12 @@
 
 @web.memoize
 def get_ol(servername=None):
-    ol = OpenLibrary(base_url=servername)
-    ol.autologin()
+    if os.getenv('LOCAL_DEV'):
+        ol = OpenLibrary(base_url="http://localhost:8080")
+        ol.login("admin", "admin123")
```

Comment:
@cdrini, @scottbarnes mentioned we may want to turn these creds for localhost which are also used in copydocs and make them environment variables in docker rather than coding them here. (for another PR)